src/openprocurement/api/traversal.py

```diff
@@ -20,9 +20,18 @@ def __init__(self, request):
         self.db = request.registry.db
 
 
+def generate_plural_name(name):
+    if name[-1] == 's':
```

Comment:
[from python import built-in](https://docs.python.org/2.7/library/stdtypes.html#str.endswith)